tests/generic_config_updater/test_dynamic_acl.py

```diff
@@ -0,0 +1,1251 @@
+        vlan_name = list(mg_facts['minigraph_vlans'].keys())[0]
+        # Use VLAN MAC as router MAC on dual-tor testbed
+        router_mac = rand_selected_dut.get_dut_iface_mac(vlan_name)
+        is_dualtor = True
+    else:
+        router_mac = rand_selected_dut.facts['router_mac']
+
+    res = rand_selected_dut.shell('cat /sys/class/net/{}/address'.format(vlan_name))
+    v4_vlan_mac = res['stdout']
+
+    list_ports = mg_facts["minigraph_vlans"][vlan_name]["members"]
+    switch_loopback_ip = mg_facts['minigraph_lo_interfaces'][0]['addr']
+
+    # Get all vlan ports
+    vlan_ports = list(mg_facts['minigraph_vlans'].values())[0]['members']
+
+    for port in vlan_ports:
+        if port in mg_facts['minigraph_port_name_to_alias_map']:
+            break
+        else:
+            continue
+    block_src_port = port
+
+    unblocked_src_port = vlan_ports[1]
+    scale_ports = vlan_ports[:]
+    block_src_port_indice = mg_facts['minigraph_ptf_indices'][block_src_port]
+    block_src_port_alias = mg_facts['minigraph_port_name_to_alias_map'][block_src_port]
+    unblocked_src_port_indice = mg_facts['minigraph_ptf_indices'][unblocked_src_port]
+    scale_ports_indices = [mg_facts['minigraph_ptf_indices'][port_name] for port_name in scale_ports]
+    # Put all portchannel members into dst_ports
+    dst_port_indices = []
+    for _, v in mg_facts['minigraph_portchannels'].items():
+        for member in v['members']:
+            dst_port_indices.append(mg_facts['minigraph_ptf_indices'][member])
+
+    # Generate destination IP's for scale test
+    scale_dest_ips = {}
+    for i in range(1, 75):
+        ipv4_rule_name = "FORWARD_RULE_" + str(i)
+        ipv6_rule_name = "V6_FORWARD_RULE_" + str(i)
+        ipv4_address = DST_IP_FORWARDED_SCALE_PREFIX + str(i)
+        ipv6_address = DST_IPV6_FORWARDED_SCALE_PREFIX + str(i) + "::1"
+        scale_dest_ips[ipv4_rule_name] = ipv4_address
+        scale_dest_ips[ipv6_rule_name] = ipv6_address
+
+    vlan_ips = {}
+
+    for vlan_interface_info_dict in mg_facts['minigraph_vlan_interfaces']:
+        if netaddr.IPAddress(str(vlan_interface_info_dict['addr'])).version == 6:
+            vlan_ips["V6"] = vlan_interface_info_dict['addr']
+        elif netaddr.IPAddress(str(vlan_interface_info_dict['addr'])).version == 4:
+            vlan_ips["V4"] = vlan_interface_info_dict['addr']
+
+    config_facts = rand_selected_dut.config_facts(host=rand_selected_dut.hostname, source="running")['ansible_facts']
+
+    vlans = config_facts['VLAN']
+    topology = tbinfo['topo']['name']
+    dut_mac = ''
+    for vlan_details in list(vlans.values()):
+        if 'dualtor' in topology:
+            dut_mac = vlan_details['mac'].lower()
+        else:
+            dut_mac = rand_selected_dut.shell('sonic-cfggen -d -v \'DEVICE_METADATA.localhost.mac\'')["stdout_lines"][0]
+        break
+
+    # Obtain uplink port indicies for this DHCP relay agent
+    uplink_interfaces = []
```

Comment:
The dst_ports for DHCP packet should be the same with the regular packets. The dst_ports should be the T1 facing ports.